CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Icinga DB Changelog
 
+## 1.1.1 (2023-08-09)
+
+This release fixes a few crashes in the Icinga DB daemon, addresses some shortcomings in the database schema,
+and makes the `icingadb-migrate` tool handle malformed events and other edge-cases more reliably.
+
+* Fix a possible crash when the Icinga 2 heartbeat is lost. #559
+* Retry additional non-fatal database errors. #593 #583
+* Make heartbeat compatible with Percona XtraDB Cluster. #609
+* Write a hint for empty arrays/dicts into `customvar_flat` for Icinga DB Web. #601
+* Schema: Allow longer names for notification objects. #584
+* Schema: Add missing indices to `hostgroup`, `servicegroup`, and `customvar_flat`. #616 #617
+* Schema: Change sort order of history event types. #626
+* `icingadb-migrate`: Ignore events that miss crucial information. #551
+* `icingadb-migrate`: Fix a foreign key error for flapping history with `ido.from` set. #554
+* `icingadb-migrate`: Fix a constraint violation for flexible downtimes that never started. #623
+* Config: Warn about unknown options (`icingadb-migrate` rejects them). #605 #631
```

Comment:
Apropos, that could even include this one, but just specifically for the migration.